LabJavaIO/src/main/java/ch/heigvd/res/labio/impl/Application.java

```diff
@@ -133,7 +132,18 @@ void clearOutputDirectory() throws IOException {
    * @throws IOException 
    */
   void storeQuote(Quote quote, String filename) throws IOException {
-    throw new UnsupportedOperationException("The student has not implemented this method yet.");
+    String tmp = new String();
```

Comment:
I detect that this code is problematic. According to the [Performance (PERFORMANCE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#performance-performance), [Dm: Method invokes inefficient new String() constructor (DM_STRING_VOID_CTOR)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#dm-method-invokes-inefficient-new-string-constructor-dm-string-void-ctor).
Creating a new java.lang.String object using the no-argument constructor wastes memory because the object so created will be functionally indistinguishable from the empty string constant "".  Java guarantees that identical string constants will be represented by the same String object.  Therefore, you should just use the empty string constant directly.

Reply:
Fixed it, the string is now initialized with empty string